OrbitCore/LinuxEventTracer.cpp

```diff
@@ -172,13 +185,59 @@ void LinuxEventTracer::Run(bool* a_ExitRequested)
                             CS.m_ThreadId = context_switch.NextTID();
                             CS.m_Time = context_switch.Timestamp();
                             CS.m_ProcessorIndex = context_switch.CPU();
-                            //TODO: Is this correct?
+                            // TODO: Is this correct?
                             CS.m_ProcessorNumber = context_switch.CPU();
                             GCoreApp->ProcessContextSwitch( CS );
                         }
                     }
                     break;
 
+                case PERF_RECORD_FORK:
```

Comment:
we might consider using cpuwide profiling in which case this even does not need to be handled.

Reply:
I created b/148209993 and added a TODO.